test/src/org/exist/xquery/functions/validate/JingOnvdlTest.java

```diff
@@ -103,6 +104,7 @@ public void onvdl_valid() throws XPathException, IOException, XpathException, SA
         executeAndEvaluate(query,"valid");
     }
 
+    @Ignore("Can cause a deadlock in txn-thread branch")
```

Comment:
:-(

Reply:
Okay so I removed those @Ignores now as we fixed it here - https://github.com/eXist-db/exist/pull/1908